upi/vsphere/main.tf

```diff
@@ -5,6 +5,17 @@ provider "vsphere" {
   allow_unverified_ssl = true
 }
 
+provider "nsxt" {
+  host                     = "${var.nsx_manager}"
+  username                 = "${var.nsx_user}"
+  password                 = "${var.nsx_password}"
+  allow_unverified_ssl     = true
+  max_retries              = 10
+  retry_min_delay          = 500
+  retry_max_delay          = 5000
+  retry_on_status_codes    = [429]
```

Comment:
It Might be worth retrying on 503 and 429

Reply:
added